models/project.php

```diff
@@ -1499,8 +1499,11 @@ public function GetSubProjectGroups()
     public function ConvertToJSON()
     {
         $response = array();
-
-        foreach (get_object_vars($this) as $k => $v) {
```

Comment:
based on PHP's documentation this should not be necessary, but for some reason get_object_vars is spitting out $PDO.